SalesforceCommerceCloud/pwa-kit
# Replace watch with nodemon

# Description
This PR attempted to find an alternative library to replace watch library what has a high vulnerability. Nodemon seems to good one to use.

https://github.com/remy/nodemon

> nodemon is a tool that helps develop Node.js based applications by automatically restarting the node application when file changes in the directory are detected.



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Add nodemon into dependency 
- Remove watch as dependency

# How to Test-Drive This PR

- Run npm run build:watch to start file watching for pwa-kit-* or commerce-sdk-react
- Make change in those library and observe that the package is re-built when a change is saved

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
